### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### LINKED ISSUE #61: Remove version pin for `greenlet` package
# 💡 Summary #

We need to remove the version pin constraint for the [`greenlet` package]() that was added in https://github.com/cisagov/ansible-role-ncats-webd/pull/62.

## Motivation and context ##

This version pin was added to allow successful installation on Debian Stretch. Once https://github.com/cisagov/skeleton-ansible-role/pull/124 is merged and trickles down it is likely no longer needed as the failure is due to the old version of `gcc` on Debian Stretch.

## Acceptance criteria ##

- [x] Support for Debian Stretch is dropped
- [x] Version pin is removed
- [x] Tests pass

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role-with-test-user.git`](https://github.com/cisagov/skeleton-ansible-role-with-test-user.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-ncats-webd.git ansible-role-ncats-webd
    cd ansible-role-ncats-webd
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role-with-test-user.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md meta/main.yml molecule/default/molecule-no-systemd.yml requirements-test.txt 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Switch back to using the default branch of [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core) once cisagov/ansible-role-cyhy-core#72 is merged.

---

> **Note**
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by jsf9k** [comment]
@jmorrowomni - This conflicted PR should be resolved in a similar manner to cisagov/ansible-role-cyhy-archive#50.  Just let me know if you prefer to hand this one off to me.

File: tasks/main.yml
Comment (jsf9k) [inline tasks/main.yml:3]: Let's go ahead and remove these unnecessary lines:
```suggestion
```

File: README.md
Comment (dav3r) [inline README.md:66]: Change of verb tense:
```suggestion
| ncats_webd_maxmind_license_key | The MaxMind license key that provides access to a GeoIP2 database subscription. | n/a | Yes |
```

**Review by jsf9k — APPROVED** [review]
I made one minor suggestion.  Please also note that:
- I added a Pre-Merge requirement to wait until a cisagov/ansible-role-cyhy-core role is merged before merging this PR.
- I set #61 to be closed by this PR.

In the future try to separate the changes into several smaller commits, and especially separate the "merge" commit from the others.  For example, in this case the removal of the greenlet pin magically appears in the merge commit but it should really be in its own commit.  See cisagov/ansible-role-cyhy-archive#50 for an example of what I'm talking about.

File: tasks/main.yml
Comment (jmorrowomni) [reply tasks/main.yml:3]: > For example, in this case the removal of the greenlet pin magically appears in the merge commit but it should really be in its own commit. See https://github.com/cisagov/ansible-role-cyhy-archive/pull/50 for an example of what I'm talking about.

I try to do that(see https://github.com/cisagov/ansible-role-cobalt-strike/pull/79).  I remember making a commit for greenlet, but I think I ended up starting that PR from scratch because of an issue I was having and must've done it at the same time the second go round

**Review by dav3r — APPROVED** [review]

File: tasks/main.yml
Comment (jmorrowomni) [reply tasks/main.yml:3]: > Let's go ahead and remove these unnecessary lines:

I saw that in another repo but wasn't sure if the extra line got deleted accidentally.  I'll make sure I remove them in other repos I see them in as well

File: README.md
Comment (jmorrowomni) [reply README.md:66]: @dav3r Good catch on my typo

**Comment by jmorrowomni** [comment]
> * I added a Pre-Merge requirement to wait until a cisagov/ansible-role-cyhy-core role is merged before merging this PR.

Just to be sure I understand properly, once we merge cisagov/ansible-role-cyhy-core we'll want to go back into this codeb ase and remove the reference to the lineage/skeleton branch before merging as well, correct?

**Comment by jsf9k** [comment]
> > * I added a Pre-Merge requirement to wait until a cisagov/ansible-role-cyhy-core role is merged before merging this PR.
> 
> Just to be sure I understand properly, once we merge cisagov/ansible-role-cyhy-core we'll want to go back into this codebase and remove the reference to the lineage/skeleton branch before merging as well, correct?

Yes, that is correct.

**Pull request merged**
